spitfire/runtime/udn_test.py

```diff
@@ -63,6 +66,13 @@ def test_in_search_list_object(self):
         self.assertIn('boom', template.placeholder_cache)
         self.assertEqual(udn.resolve_placeholder('boom', template, None), 'bam')
 
+    def test_method_cycle_elimination(self):
```

Comment:
This test passed before your changes too right?  It'd be good to have test that fails at head and passes here